README.md

```diff
@@ -204,7 +204,7 @@ The current version of Lumberjack requires:
 - watchOS 4 or later
 
 ### Backwards compatibility
-- for Xcode 12 and Swift up to 5.4, use the 3.7.4 version
+- for iOS 9, iOS 10, OS X 10.10, OS X 10.11, macOS 10.12, Xcode 12, Xcode 13 and Swift up to 5.4, use the 3.7.4 version
```

Comment:
Can we simplify this by listing the upper limit?

```suggestion
- for iOS/tvOS up to 10, watchOS up to 4, macOS up to 10.12, Xcode up to 13 and Swift up to 5.4, use the 3.7.4 version
```

Reply:
Simple is better. :)